embodiedscan/structures/bbox_3d/euler_box3d.py

```diff
@@ -1,43 +1,24 @@
-# Copyright (c) OpenMMLab. All rights reserved.
+# Copyright (c) OpenRobotLab. All rights reserved.
 import numpy as np
 import torch
 from pytorch3d.ops import box3d_overlap
 from pytorch3d.transforms import euler_angles_to_matrix, matrix_to_euler_angles
 
+from ..points.base_points import BasePoints
 from .base_box3d import BaseInstance3DBoxes
-from .base_points import BasePoints
 from .utils import rotation_3d_in_euler
 
 
 class EulerInstance3DBoxes(BaseInstance3DBoxes):
-    """3D boxes of instances in Depth coordinates.
+    """3D euler boxes.
```

Comment:
3D boxes with 1-D orientation represented by three Euler angles.

Reply:
This box serves as the base class for general 3D bounding box predictions. The original 7-DoF box definition in MMDet3D is replaced and deprecated. We only temporarily keep the operations related to the point coordinate system in its child class, such as the flipping augmentation.